src/main/java/dev/jbang/util/Util.java

```diff
@@ -99,6 +101,16 @@ public class Util {
 	private static Boolean downloadSources;
 	private static Instant startTime = Instant.now();
 
+	private static final StringSubstitutor urlSubstitutor = createUrlSubstitutor();
```

Comment:
we have `PropertiesValueResolver` to do property replacement. don't think there is a need to use different one here?

Reply:
Done